pkg/size/size.go

```diff
@@ -0,0 +1,187 @@
+package size
```

Comment:
why do we require this package?
support only bytes as input, then we don't need this package.

Reply:
This is a nice package, but I agree with @Madhu-1 API only need to support bytes

Reply:
@aravindavk @Madhu-1 form gluster-block info we get the response in string format, we need to parse the string representation, so I thought a separate package will be helpful in future also.